app/v1/connectors/httpparsers/HttpParser.scala

```diff
@@ -47,7 +47,8 @@ trait HttpParser {
 
   def retrieveCorrelationId(response: HttpResponse): String = response.header("CorrelationId").getOrElse("")
 
-  private val multipleErrorReads: Reads[List[DesErrorCode]] = (__ \ "failures").read[List[DesErrorCode]]
+  // Not required while multiple errors are not supported
+  //private val multipleErrorReads: Reads[List[DesErrorCode]] = (__ \ "failures").read[List[DesErrorCode]]
```

Comment:
What does existing VAT do if we are returned multiple errors?

Reply:
InternalServerError